ios/help/about/whatsnew.md

```diff
@@ -2,6 +2,6 @@
 title: What's New
 ---
 
-Here are some of the new features we have added to Keyman for iPhone and iPad 16.0:
+Here are some of the new features and major bug fixes we have added to Keyman for iPhone and iPad 16.0:
```

Comment:
Right, I forgot to mention:  that comment was meant for each page, not just the one iOS one.  It would definitely apply to at least one of the other lists.